src/main/scala/dpla/ingestion3/reports/PropertyValueReport.scala

```diff
@@ -155,14 +155,6 @@ class PropertyValueReport (
             value = extractValue(oreAggregation.sourceResource.place)
           )
         })
-      case "sourceResource.place.name" =>
```

Comment:
This was simply removing a duplicate report entry for `sourceResource.place.name`